app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -35,6 +29,10 @@ expander:
   SKE-bursaries:
     title: SKE bursaries
     text: Most bursaries are only for UK citizens. However, you may be eligible for an SKE bursary if you're training to teach physics or languages.
+cta_arrow_link:
+  maths:
+    link_target: "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
```

Comment:
This isn't the right link!

But I'm also not sure why we are pointing to maths but not any of the other subjects - feels a bit odd?

Would it be better to point to the explore subjects page?